docs/mcd.md

```diff
@@ -114,8 +114,51 @@ The connector will not load if there are circular references.
 | field | Name of the field whose value will be checked for equality | No | Any string value | |
 | value | The value to match. If the field's value is equal to this, the condition is true. | No | Any string value | |
 
+## Connection Field Platform Integration 
 
-## Example - A Non-Editable Field
+As referenced above in the `<field>` element section, some `name` attribute values describe platform functionality. It is important that if the connector functionality desired matches any of the descriptions below then names and values below must be used.
```

Comment:
nits: consider 
"connector functionality desired" -> "desired connector functionality"
"then names" -> "then the names"